core/test/docs.toml

```diff
@@ -37,9 +39,9 @@ timing = false     # optional, whether to log debug timing statements
 
 [results]
 # These results files are always written
-flow = "results/flow.arrow"       # optional, default "results/flow.arrow"
-basin = "results/basin.arrow"     # optional, default "results/basin.arrow"
-control = "results/control.arrow" # optional, default "results/control.arrow"
+flow = "results/flow.arrow"             # optional, default "results/flow.arrow"
```

Comment:
Why do you still have to specify "results" here? :)

Shouldn't this just be flow.arrow now?

Reply:
Yeah probably, but it might be good to have that discussion first before changing it. Results are out of scope for this PR?

Reply:
But you introduced results_dir.

Shouldn't this part be adapted accordingly?

Or am I misunderstanding what results_dir does?

Reply:
This PR does not introduce results_dir functionality. I added it to the docs.toml because it was missing there on request of @visr, and autoformatting did a bit more to that file. The Python code does not (yet) read results files.

Reply:
I see.
If this is unrelated to your PR then let's merge it.
Let's remember to discuss that result_dir thing on Monday :)